addons/mail/static/tests/discuss/call/call_tests.js

```diff
@@ -105,16 +105,6 @@ QUnit.test("should disconnect when closing page while in call", async (assert) =
 });
 
 QUnit.test("should display invitations", async (assert) => {
-    patchWithCleanup(browser, {
-        Audio: class extends Audio {
-            pause() {
-                assert.step("pause_sound_effect");
-            }
-            play() {
-                assert.step("play_sound_effect");
-            }
-        },
-    });
```

Comment:
Note that changing this test was not mandatory to make the test pass, but it was poorly written by asserting it triggers play/pause of any sound, which was unclear when this test failed.

I adapted it so it more explicitly asserts the playing of `incoming-call`